reccmp/isledecomp/compare/db.py

```diff
@@ -10,27 +10,37 @@
 from reccmp.isledecomp.types import EntityType, ImageId
 
 _SETUP_SQL = """
+    CREATE TABLE names (
+        img integer not null,
+        addr integer not null,
+        name text,
+        computed_name text,
+        primary key (img, addr)
+    );
+
     CREATE TABLE entities (
         orig_addr int unique,
         recomp_addr int unique,
-        kvstore text default '{}',
-        ref_orig integer as (json_extract(kvstore, '$.ref_orig')),
-        ref_recomp integer as (json_extract(kvstore, '$.ref_recomp'))
+        kvstore text default '{}'
+    );
+
+    CREATE TABLE refs (
+        img integer not null,
+        addr integer not null,
+        ref integer not null,
+        disp0 integer not null default 0,
+        disp1 integer not null default 0,
+        primary key (img, addr)
     );
 
-    CREATE VIEW orig_refs (orig_addr, ref_id, nth) AS
-        SELECT thunk.orig_addr, ref.rowid,
-            Row_number() OVER (partition BY thunk.ref_orig order by thunk.orig_addr) nth
-        FROM entities thunk
-        INNER JOIN entities ref on thunk.ref_orig = ref.orig_addr
-        WHERE json_extract(thunk.kvstore, '$.vtordisp') IS NULL;
+    CREATE VIEW matches (match_id, addr_x, addr_y) AS
```

Comment:
Is there a reason to use the names `addr_x` and `addr_y` instead of `orig_addr` and `recomp_addr`? If there isn't, I'd stick to the established names for consistency.